features/ptp/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
AFAIK, there are no yaml files here to test anything

Reply:
done